Formula/agda.rb

```diff
@@ -109,60 +81,114 @@ def caveats
 
     if build.with? "stdlib"
       s += <<-EOS.undent
-      To use the Agda standard library, point Agda to the following include dir:
-        #{HOMEBREW_PREFIX}/lib/agda/src
-      EOS
-
-      if build.with? "malonzo"
-        s += <<-EOS.undent
+      Agda 2.5.1 includes new library management features.
+      http://agda.readthedocs.org/en/stable/tools/package-system.html
```

Comment:
Sorry, I thought I'd said that already and I've already explained that caveats are for Homebrew-specific information.